language-python/language-python.cabal

```diff
@@ -41,9 +42,10 @@ Library
       pretty == 1.1.*,
       array >= 0.4 && < 0.6,
       transformers >= 0.3 && < 0.7,
-      monads-tf == 0.1.*,
+      monads-tf == 0.3.*,
       utf8-string >= 1 && < 2
-   build-tools: happy, alex
+   build-tools: happy >= 1.20, alex
+   build-tool-depends: happy:happy
```

Comment:
One should not have both `build-tools` (the old, deprecated way) and `build-tool-depends`.